include/hobbes/lang/module.H

```diff
@@ -164,6 +188,8 @@ template <typename T>
     virtual T with(const MVarDef*)     const = 0;
     virtual T with(const ClassDef*)    const = 0;
     virtual T with(const InstanceDef*) const = 0;
+    virtual T with(const MUnsafePragmaDef*)  const = 0;
```

Comment:
Doesn't this seem a little overly specific for this level of definition?  If you wanted to support custom pragmas (don't we already have something for this?) you wouldn't want to add a new type for each one here right?